servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -246,6 +247,8 @@ def query_jobs(body, **kwargs):
     auth = kwargs.get('auth')
     headers = kwargs.get('auth_headers')
     query = QueryJobsRequest.from_dict(body)
+    if query.labels is not None:
```

Comment:
It might be nice to make `_format_query_labels` resilient to empty input instead of protecting it at each call site (which, granted, is only one right now)

Reply:
done